### ISSUE TITLE
Spell Attack Bonus Not Updating When Using Multiple Active Effects

### ISSUE THREAD

**Issue by emphaticrain** [sdenec/tidy5e-sheet#592]
Using Tidy 5e v0.5.33 on Foundry 9.280.

It seems that when using multiple active effects to enhance a character's Spell Attack Bonus, the display at the bottom left of the Spellbook tab does not update correctly. The magic number, it seems, is four modifications to the d20 test, three of which are already present (using Better Rolls module, though I don't think this is impacting anything here): 1d20 + 0 + Spellcasting Ability Mod + Proficiency Bonus.

Adding a single instance of a SAB modifier, like a Wand of the War Mage +1, updates correctly on the SAB display on the Spellcasting tab. However, add another one, say a Robe of the Archmagi (which grants a +2 bonus to spell attack), and the SAB display remains the same, as if the Robe wasn't there. But roll a spell attack, and the mods are all there and the math works out correctly.

I have a lot of modules, but I'm not sure what would be restricting the character sheet from updating when everything else works fine. This is likely only a problem if a player wanted to roll real clicky-clacky dice for a session and used the SAB to calculate their attack rolls wrong, but it seems like a bug all the same. Thanks!

**Comment by sdenec** [comment]
Hm, I had to fix something with roll formulas - especially regarding bonuses that are varible (e. g. +1d4) - for v10. Maybe there are conflicts with mod introduced bonuses or formulas that prevent the calculation to give the proper result. But in any case - the actual roll will always be correct as this is a different calculation system than the simple value display I tried to implement.

On another note - since v9 is somewhat deprecated and I already have trouble maintaining this one v10 branch I'm sorry to tell you that I won't be able to look further into this issue :( v0.5.33 is the last version for v9, very sorry.
